commons/passive/garmin/garmin_generic_accelerometer.avsc

```diff
@@ -0,0 +1,46 @@
+{
+   "namespace":"org.radarcns.passive.garmin",
+   "type":"record",
+   "name":"GarminGenericAccelerometer",
+   "doc":"Data from accelerometer sensor.",
+   "fields":[
+      {
+
+         "name":"time",
+         "type":"double",
+         "doc":"Device timestamp in UTC (s)."
+      },
+       {
+
+               "name":"timeReceived",
+               "type":"double",
+               "doc":"Device receiver timestamp in UTC (s)."
+            },
+      {
+         "name":"timestamp13",
+         "type":"double",
+         "doc":"A timestamp13 value which is a 13 bit local time, the time when this sample was gathered from the Garmin device (useful for calculating jitter in the streaming data)."
```

Comment:
I'm still not sure how this timestamp can be used.

Is the unit seconds? What is the range and precision (millis? seconds?) Since there are only 13 bits of information, will there be rollover after some time (e.g. after 2^13 seconds, this timestamp reverts to 0?). Some suggestions then:
call this `timeIndex`. If possible, use the `int` type here and put the corresponding unit in the docs (so, if this displays milliseconds, use milliseconds, otherwise, seconds). If the 13 bit local time indicates a fraction (e.g. 1/256 seconds), then put that as the unit.

Reply:
Got confirmation from Garmin team that this 13 bit time stamp is deprecated.  It was originally necessary due to an issue with timestamp alignment in older SDK versions.  That issue has been corrected, and we are now able to provide millisecond precision timestamps for accelerometer samples using the getMillisecondTimestamp()​ method.  The 13 bit timestamp is still calculated and provided as a convenience for partners that designed against the older solution. Will be removed from the schema definition.